core/src/main/resources/xsl/builder.xsl

```diff
@@ -53,22 +53,30 @@
     <xsl:param name="user" as="xsd:string" select="'unknown'" />
     <xsl:param name="creator" as="xsd:string" select="'unknown'"/>
     <xsl:param name="configMetadata" as="node()">
-        <meta>
-            <config option="checkXSDGrammar"  value="false"/>
-            <config option="checkJSONGrammar" value="false"/>
-            <config option="checkWellFormed"  value="false"/>
-            <config option="checkElements"    value="false"/>
-            <config option="checkPlainParams" value="false"/>
-            <config option="checkHeaders"     value="false"/>
+        <prams>
+          <meta>
+            <config option="enableIgnoreJSONSchemaExtension" value="true"/>
             <config option="setParamDefaults" value="false"/>
+            <config option="enableMessageExtension" value="true"/>
+            <config option="joinXPathChecks" value="false"/>
             <config option="doXSDGrammarTransform" value="false"/>
-            <config option="enablePreProcessExtension" value="false"/>
-            <config option="enableIgnoreXSDExtension" value="false"/>
-            <config option="enableIgnoreJSONSchemaExtension" value="false"/>
-            <config option="enableMessageExtension" value="false"/>
+            <config option="enablePreProcessExtension" value="true"/>
+            <config option="removeDups" value="false"/>
+            <config option="checkXSDGrammar" value="false"/>
+            <config option="enableCaptureHeaderExtension" value="true"/>
+            <config option="xpathVersion" value="1"/>
+            <config option="checkPlainParams" value="false"/>
+            <config option="checkWellFormed" value="false"/>
+            <config option="enableAnyMatchExtension" value="true"/>
+            <config option="enableIgnoreXSDExtension" value="true"/>
+            <config option="checkJSONGrammar" value="false"/>
+            <config option="checkElements" value="false"/>
+            <config option="preserveRequestBody" value="false"/>
+            <config option="checkHeaders" value="true"/>
             <config option="enableRaxRolesExtension" value="false"/>
-            <config option="enableCaptureHeaderExtension" value="false"/>
-        </meta>
+            <config option="maskRaxRoles403" value="false"/>
+          </meta>
+        </prams>
     </xsl:param>
 
     <xsl:variable name="checkXSDGrammar"  as="xsd:boolean" select="xsd:boolean(check:optionValue($configMetadata, 'checkXSDGrammar'))"/>
```

Comment:
Does the `check:optionValue` function know to flatten out the new `prams`/`params`?